.github/workflows/test.yml

```diff
@@ -6,119 +6,119 @@ on:
-        #   # run: curl -L https://github.com/vegaprotocol/vegawallet/releases/download/v${{ matrix.vegawallet-version }}/vegawallet-linux-amd64.zip -O
-        # - name: ls current
-        #   run: ls
-        # - name: ls previous
-        #   run: ls ..
-        # - name: Unzip wallet
-        #   run: unzip vegawallet-linux-amd64.zip
-        - name: Download and Unzip latest wallet
-          run: curl -s https://api.github.com/repos/vegaprotocol/vegawallet/releases/latest | awk -F\" '/browser_download_url.*linux-amd64.zip/{print $(NF-1)}' | xargs wget && unzip vegawallet-linux-amd64.zip && rm vegawallet-linux-amd64.zip
-        - name: Initialize wallet
-          run: ./vegawallet init -f
-        - name: Create recovery phrase file
-          run: echo ${{ secrets.TEST_WALLET_RECOVERY }} > recovery-phrase.txt
-        - name: Create recovery phrase file
-          run: echo "dev12345" > passphrase.txt
-        - name: Import wallet
-          run: ./vegawallet import --wallet UI_TEST_1 --recovery-phrase-file recovery-phrase.txt --passphrase-file passphrase.txt
-        - name: Import config
-          run: ./vegawallet network import --from-url https://raw.githubusercontent.com/vegaprotocol/networks/master/fairground/fairground.toml
-        - name: List networks
-          run: ./vegawallet network list
-        - name: Start service
-          run: ./vegawallet service run --network fairground  &
-        - name: 'BrowserStack Env Setup'  # Invokes the setup-env action
-          uses: browserstack/github-actions/setup-env@master
-          with:
-            username: ${{ secrets.BROWSERSTACK_USER }}
-            access-key: ${{ secrets.BROWSERSTACK_KEY }}
-          env:
-            BROWSERSTACK_USER: ${{ secrets.BROWSERSTACK_USER }}
-            BROWSERSTACK_KEY: ${{ secrets.BROWSERSTACK_KEY }}
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16.14.0
+      - name: Install root dependencies
+        run: npm run yarn:install:deps
+      - name: Install
+        run: npm install
+      # - name: Download wallet
+      #   run: curl -L https://github.com/vegaprotocol/vegawallet/releases/download/v0.13.1/vegawallet-linux-amd64.zip -O
+      #   # run: curl -L https://github.com/vegaprotocol/vegawallet/releases/download/v${{ matrix.vegawallet-version }}/vegawallet-linux-amd64.zip -O
+      # - name: ls current
+      #   run: ls
+      # - name: ls previous
+      #   run: ls ..
+      # - name: Unzip wallet
+      #   run: unzip vegawallet-linux-amd64.zip
+      - name: Download and Unzip latest wallet
+        run: curl -s https://api.github.com/repos/vegaprotocol/vegawallet/releases/latest | awk -F\" '/browser_download_url.*linux-amd64.zip/{print $(NF-1)}' | xargs wget && unzip vegawallet-linux-amd64.zip && rm vegawallet-linux-amd64.zip
```

Comment:
chore (blocking): pipeline should be deterministic using latest means that it is not. Let's use the same logic as in the frontend monorepo please and fix to a matrix defined version of the wallet:

https://github.com/vegaprotocol/frontend-monorepo/blob/master/.github/workflows/cypress.yml#L16-L17

and consuming that in this curl:
https://github.com/vegaprotocol/frontend-monorepo/blob/master/.github/workflows/cypress.yml#L38-L39